crates/cext/src/transpiler/passes/remove_identity_equiv.rs

```diff
@@ -0,0 +1,101 @@
+// This code is part of Qiskit.
+//
+// (C) Copyright IBM 2025
+//
+// This code is licensed under the Apache License, Version 2.0. You may
+// obtain a copy of this license in the LICENSE.txt file in the root directory
+// of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+//
+// Any modifications or derivative works of this code must retain this
+// copyright notice, and modified files need to carry a notice indicating
+// that they have been altered from the originals.
+
+use crate::pointers::const_ptr_as_ref;
+
+use qiskit_circuit::circuit_data::CircuitData;
+use qiskit_circuit::converters::dag_to_circuit;
+use qiskit_circuit::dag_circuit::DAGCircuit;
+use qiskit_transpiler::passes::run_remove_identity_equiv;
+use qiskit_transpiler::target::Target;
+
+/// @ingroup QkTranspilerPasses
+/// Run the RemoveIdentityEquivalent transpiler pass on a circuit.
+///
+/// Removes gates whose effect is close to an identity operation up to a global phase
+/// and up to the specified tolerance. Parameterized gates are not considered by this pass.
+///
+/// For a cutoff fidelity :math:`f`, this pass removes gates whose average
+/// gate fidelity with respect to the identity is below :math:`f`. Concretely,
+/// a gate :math:`G` is removed if :math:`\bar F < f` where
+///
+/// $bar{F} = \frac{1 + d F_{\text{process}}}{1 + d},F_{\text{process}} =
+/// \frac{|\mathrm{Tr}(G)|^2}{d^2}$
+///
+/// where $d = 2^n$ is the dimension of the gate for $n$ qubits.
```

Comment:
This mixes $ and :math: -- I would assume :math: is the right one, but I'm not sure 🙂

Reply:
A good I'll update the syntax to use `\f$` to fix this and make sure the output renders correctly.